### ISSUE TITLE
EC2 Instances having error chmod 400

### ISSUE THREAD

**Issue by newbielinuxuser** [li0nel/laravel-terraform#1]
As the subject mentioned, when running `terraform apply plan.tf `according to the documentation from your guides (https://hackernoon.com/stop-manually-provisioning-aws-for-laravel-use-terraform-instead-11b8b360617c) 
This happened.
Error: Error applying plan:

1 error(s) occurred:

* module.ec2.null_resource.chmod: Error running command 'chmod 400 ./key-default': exit status 1. Output: 'chmod' is not recognized as an internal or external command,
operable program or batch file.

**Comment by newbielinuxuser** [comment]
Apparently within ec2.tf , for windows users you will have to change 
from
`command = "chmod 400 ${local.private_key_filename}"`
to
`command = "attrib +r ${local.private_key_filename}"`

cacls and xcacls not gonna work in Win10 by default. icacls will work but you will have to go through with the syntax, so I used `attrib`, that appears to have the same effect as the others. Just make sure the file is set to read-only if the private key is gonna used and transferred to other unix machine.

TL:DR
You can basically ignore or comment out that particular line (ec2.tf Line #187) if you are using windows and debug using windows.
